services/src/workflows/postgres_workflow_registry.rs

```diff
@@ -1,3 +1,5 @@
+#![cfg(feature = "postgres")]
```

Comment:
put the cfg(feature = "postgres") into mod.rs before "pub mod postgres_xxx;"

Reply:
26511745f68be89453766afe268f7d49343fa48b